### ISSUE TITLE
Request: PLease add some basic two way interaction examples

### ISSUE THREAD

**Issue by DeaconDP** [de-id/live-streaming-demo#3]
I got the basics up and running, but it would be awesome to build this out a bit more to handle two-way conversations.

**Comment by helenawsu** [comment]
Same here. Wish to see more tutorials on how to integrate chatgpt.

**Comment by Guliver2-Magic** [comment]
Yes more tutorial please about integrating ChatGpt like you did on your demo.

**Comment by devchamps** [comment]
+4 now, basic 2 way communication example please.

**Comment by Richkeyrick** [comment]
Please d-id we need this can you complete the code with  two way interaction, user chat and with answering bot
please....

**Comment by lrovaris** [comment]
please

**Comment by skyz8421** [comment]
please

**Comment by Nik395** [comment]
please

**Comment by rttombs** [comment]
i replace the image URL but it is not using it... any idea?

**Comment by emorling** [comment]
+1

**Comment by faustinopsy** [comment]
> Mesmo aqui. Gostaria de ver mais tutoriais sobre como integrar o chatgpt.
fiz uma integração com GPT e youtube
https://www.youtube.com/watch?v=_D9X9rZVGpw
